graylog2-server/src/main/java/org/graylog/events/contentpack/entities/EmailEventNotificationConfigEntity.java

```diff
@@ -50,6 +51,9 @@
     @JsonProperty(FIELD_BODY_TEMPLATE)
     public abstract ValueReference bodyTemplate();
 
+    @JsonProperty(FIELD_HTML_BODY_TEMPLATE)
+    public abstract ValueReference htmlBodyTemplate();
```

Comment:
We need to make sure that older content packs (without this field) are still installable.
I suggest you initialize the htmlBodyTemplate in the `Builder.create()`
https://github.com/Graylog2/graylog2-server/pull/9790/files#diff-455b0c407c9c4eac48cf30ae314d1e2222e29fb27954534f291d6a84b3facf87L70

`.htmlBodyTemplate(ValueReference.of(""))`